src/EditorFeatures/CSharpTest2/Recommendations/VarKeywordRecommenderTests.cs

```diff
@@ -251,5 +251,22 @@ public VarKeywordRecommenderTests()
 @"class C {
      void M1(out $$");
         }
+
+        [Fact, Trait(Traits.Feature, Traits.Features.KeywordRecommending)]
+        public async Task TestVarPatternInSwitch()
+        {
+            await VerifyKeywordAsync(AddInsideMethod(
+@"switch(o)
+    {
+        case $$
```

Comment:
why is 'var' appearing here?  i don't see what an inferred type does for you here.